src/@chakra-ui/components/Table.ts

```diff
@@ -26,7 +26,7 @@ const baseStyle = defineMergeStyles(
       borderBottom: "1px",
       borderColor: "body.base",
       color: "body.base",
-      textTransform: "capitalize",
+      textTransform: "normal",
```

Comment:
@corwintines We'd want "none" here; "normal" not recognized

<img width="351" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/e0826fd9-9a4f-49b6-a3ed-c41f887973bd">

https://www.w3schools.com/cssref/pr_text_text-transform.php
<img width="1218" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/e2a2f384-fad4-4c41-af98-d78b31b75ce6">

```suggestion
      textTransform: "none",
```

<img width="1194" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/83a8340b-c19a-48d0-9550-f29f7ead76d3">